helm/kubeblocks/falkordb-cluster-omnistrate/tests/integration/cluster/test_cluster_integration.py

```
@@ -0,0 +1,1936 @@
+        logger.info(f"  Total queries: {total}")
+        logger.info(f"  Successful: {success} ({availability_pct:.2f}%)")
+        logger.info(f"  Failed: {self.results['failed_queries']}")
+        
+        if self.results['latencies']:
+            logger.info(f"  Latency - Avg: {self.results['latency_avg']*1000:.1f}ms, "
+                       f"P50: {self.results['latency_p50']*1000:.1f}ms, "
+                       f"P95: {self.results['latency_p95']*1000:.1f}ms, "
+                       f"P99: {self.results['latency_p99']*1000:.1f}ms")
+        
+        if self.results['errors']:
+            logger.info(f"  Error summary:")
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)